assets/js/WfHamburgerMenu.js

```diff
@@ -55,7 +55,7 @@ export class WfHamburgerMenu extends React.Component {
           </DropdownToggle>
 
           <DropdownMenu>
-            <DropdownItem onClick={this.handleClickOpenLocaleSwitcher}><i className='icon icon-language' /><Trans id='js.WfHamburgerMenu.menu.language'>Language</Trans></DropdownItem>
+            <DropdownItem className='locale-switcher-show' onClick={this.handleClickOpenLocaleSwitcher}><i className='icon icon-language' /><Trans id='js.WfHamburgerMenu.menu.language'>Language</Trans></DropdownItem>
```

Comment:
Can't the integration test click on `.dropdown-item i.icon-language`?

If we absolutely need to add a class name to make integration tests work, then we need a comment explaining why. Otherwise, the reader will assume this class is for CSS.